### PR TITLE
Singularity -> Apptainer migration

### PR DESCRIPTION
**Pull request by willfurnass**
- [x] Migrate content
- [x] Section on Apptainer vs Singularity

Could add redirect from `singularity.rst` pages to `apptainer.rst` pages?  Not sure if really needed.

NB not to be merged until Apptainer installed.

### CODE REVIEW

**Comment by willfurnass** [comment]
@jkwmoore Would be grateful if you could skim through this at some point as may want to merge quickly once Apptainer lands in EPEL.

**Comment by jkwmoore** [comment]
Will do! Catching up on Easter emails at the moment but I expect to have a flip through this afternoon.

File: hpc/modules.rst [hunk 1]
```diff
@@ -35,7 +35,7 @@ If you switch to using a cluster other than Bessemer or ShARC then you will like
 Modules are not the only way of managing software on clusters: increasingly common approaches include:
 
 * the :ref:`Conda <sharc-python-conda>` package manager (Python-centric but can manage software written in any language; can be used on Bessemer and ShARC);
-* :ref:`Singularity <singularity_sharc>`, a means for deploying software in `containers <https://en.wikipedia.org/wiki/Operating-system-level_virtualization>`__ (similar to `Docker <https://www.docker.com/>`__; can be used on ShARC).
+* :ref:`Apptainer/Singularity <apptainer_sharc>`, a means for deploying software in `containers <https://en.wikipedia.org/wiki/Operating-system-level_virtualization>`__ (similar to `Docker <https://www.docker.com/>`__; can be used on ShARC).
```
[the remark this answers is not in the document]
Comment (willfurnass) [reply hpc/modules.rst]: Done

**Comment by willfurnass** [comment]
@jkwmoore Thanks for the review!  I'll park this PR for now and set its status to 'draft' until an Apptainer RPM hits EPEL.

**Comment by jkwmoore** [comment]
> @jkwmoore Thanks for the review! I'll park this PR for now and set its status to 'draft' until an Apptainer RPM hits EPEL.

Not quite done yet as I am going in folder/file order :) might be a few more bits in a sec.

**Comment by jkwmoore** [comment]
Finished on my end, LGTM!

**Comment by willfurnass** [comment]
Also, note that there are some more general questions about whether the centrally-provided Singularity images should be advertised or have been properly deprecated, but I think it makes sense to tackle those via separate Issues/PRs.

**Pull request merged**